packages/integrations/image/components/Picture.astro

```diff
@@ -41,7 +41,7 @@ const {
 	fit,
 	background,
 	position,
-	formats = ['avif', 'webp'],
+	formats = ['webp', 'avif'],
```

Comment:
I think we should only change the defaults when the user pass in a GIF, similar to https://github.com/withastro/astro/issues/5028#issuecomment-1272390635

This also won't close #5028 as you would need sharp to process GIFs, we should document this separately too for GIF handling if it's not possible to support ootb.

Reply:
Actually, The GIF can be handled by the `squoosh`.
![image](https://user-images.githubusercontent.com/27261522/201805328-4cf36bd0-181b-4033-b815-65d8159f0107.png)
look like this GIF picture was changed into a long picture.

Reply:
@bluwy hello places looking at if you are available

Reply:
It's working for me only when I use sharp. Squoosh gives me `Buffer has an unsupported format` in my vscode terminal, and the network request showed a 500 in dev.

I'm still confused what this fixes here. It would be great if you can write some tests for this to verify it.

Reply:
> It's working for me only when I use sharp. Squoosh gives me `Buffer has an unsupported format` in my vscode terminal, and the network request showed a 500 in dev.
> 
> I'm still confused what this fixes here. It would be great if you can write some tests for this to verify it.

There is not a error with squoosh In my computer

Reply:
here's a repro: https://stackblitz.com/edit/github-lmp1qc-tcfjrz?file=astro.config.mjs,src%2Fpages%2Findex.astro

Reply:
I got it , I replicated this bug ,thanks ,I will fix it